client/my-sites/sidebar/manage-menu.jsx

```diff
@@ -101,6 +101,18 @@ class PublishMenu extends PureComponent {
 				showOnAllMySites: false,
 			} );
 		}
+
+		items.push( {
+			name: 'discussions',
+			label: this.props.translate( 'Discussions' ),
+			capability: 'edit_posts', // TODO: should be 'moderate_comments'
+			queryable: true,
+			config: 'comments/management',
```

Comment:
It's not clear to me how the feature flag checks work here.
We have a `config: 'feature'` property in each sidebar item, yet in at least one case we also check for `config.isEnabled` before pushing the item to the list.

Maybe we should add a `config.isEnabled( 'comments/management' )` check to avoid showing it to everybody while the section is still hidden or, probably the better choice, remove it from the `manage/media` item?